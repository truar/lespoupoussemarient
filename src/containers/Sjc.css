.Sjc {
    background: url('../images/sjc.jpg') no-repeat center fixed;
    background-size: cover; /* version standardisÃ©e */
    height: 100vh;
}

.Sjc>.WrapperContent {
    flex-grow: 1;
}

.Sjc>.WrapperContent>.WrapperContent {
    flex-grow: 0.1;
}

.Sjc ul {
    text-align: left;
}

.Sjc ul li {
    list-style: none;
}

.Sjc>.WrapperContent>.WrapperContent:nth-child(2) {
    width: 85%;
    height: 40vh;
}

@media(min-width: 768px) {
    .Sjc>.WrapperContent {
        flex-direction: row;
    }

    .Sjc>.WrapperContent>.WrapperContent {
        flex-grow: 0;
    }
    
    .Sjc>.WrapperContent>.WrapperContent {
        height: 40vh;
    }

    .Sjc>.WrapperContent>.WrapperContent:nth-child(2) {
        width: 40%;
    } 
}