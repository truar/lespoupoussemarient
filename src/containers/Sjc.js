import React, { Component } from 'react';
import PageTitle from '../components/PageTitle';
import Section from '../components/Section';
import { withScriptjs, withGoogleMap, GoogleMap, Marker } from "react-google-maps";
import WrapperContent from '../components/WrapperContent';
import EnvVariables from '../libs/EnvVariables';
import 'open-iconic/font/css/open-iconic-bootstrap.min.css';
import './Sjc.css';

class Sjc extends Component {
    render() {
        const MyMapComponent = withScriptjs(withGoogleMap((props) =>
            <GoogleMap
                defaultZoom={10}
                defaultCenter={{ lat: 45.304472, lng: 5.510262 }}
            >
                {props.isMarkerShown && <Marker position={{ lat: 45.304472, lng: 5.510262 }} />}
            </GoogleMap>
        ));
        const GOOGLE_API_KEY = process.env.REACT_APP_GOOGLE_API_KEY || 'mySecretKey';
        const googleMapUrl = `https://maps.googleapis.com/maps/api/js?key=${GOOGLE_API_KEY}`;
        return (
            <Section id="Sjc" className="Sjc">
                <PageTitle title="Saint-Jean de Chépy" />
                <WrapperContent>
                    <WrapperContent>
                        <WrapperContent>
                            <h3>Adresse :</h3>
                            <ul className="adress">
                                <li>Domaine St-Jean de Chépy</li>
                                <li>11 Chemin du Domaine</li>
                                <li>38210 Tullins</li>
                            </ul>
                        </WrapperContent>
                        <WrapperContent>
                            <h3>A proximit&eacute; :</h3>
                            <ul>
                                <li>Gare TGV <span className="oi oi-arrow-thick-right"></span> Grenoble</li>
                                <li>Aéroport <span className="oi oi-arrow-thick-right"></span> Lyon-Saint-Ex</li>
                                <li>Autoroutes <span className="oi oi-arrow-thick-right"></span> A48 et A49</li>
                            </ul>
                        </WrapperContent>
                    </WrapperContent>
                    <WrapperContent>
                        <MyMapComponent
                            isMarkerShown
                            googleMapURL={googleMapUrl}
                            loadingElement={<div style={{ height: '100%' }} />}
                            containerElement={<div style={{ width: '100%', height: '100%' }} />}
                            mapElement={<div style={{ height: '100%' }} />}
                            />
                    </WrapperContent>
                </WrapperContent>
            </Section>
        );
    }
}

export default Sjc;