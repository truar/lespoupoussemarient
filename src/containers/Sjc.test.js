import React from 'react';
import Sjc from './Sjc';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const component = shallow(<Sjc />);
  expect(component).toMatchSnapshot();
});

