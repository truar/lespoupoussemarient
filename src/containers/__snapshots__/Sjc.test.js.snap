// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Sjc />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <PageTitle
          title="Saint-Jean de Chépy"
        />,
        <WrapperContent>
          <WrapperContent>
            <WrapperContent>
              <h3>
                Adresse :
              </h3>
              <ul
                className="adress"
              >
                <li>
                  Domaine St-Jean de Chépy
                </li>
                <li>
                  11 Chemin du Domaine
                </li>
                <li>
                  38210 Tullins
                </li>
              </ul>
            </WrapperContent>
            <WrapperContent>
              <h3>
                A proximité :
              </h3>
              <ul>
                <li>
                  Gare TGV 
                  <span
                    className="oi oi-arrow-thick-right"
                  />
                   Grenoble
                </li>
                <li>
                  Aéroport 
                  <span
                    className="oi oi-arrow-thick-right"
                  />
                   Lyon-Saint-Ex
                </li>
                <li>
                  Autoroutes 
                  <span
                    className="oi oi-arrow-thick-right"
                  />
                   A48 et A49
                </li>
              </ul>
            </WrapperContent>
          </WrapperContent>
          <WrapperContent />
        </WrapperContent>,
      ],
      "className": "Sjc",
      "id": "Sjc",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "title": "Saint-Jean de Chépy",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <WrapperContent>
              <WrapperContent>
                <h3>
                  Adresse :
                </h3>
                <ul
                  className="adress"
                >
                  <li>
                    Domaine St-Jean de Chépy
                  </li>
                  <li>
                    11 Chemin du Domaine
                  </li>
                  <li>
                    38210 Tullins
                  </li>
                </ul>
              </WrapperContent>
              <WrapperContent>
                <h3>
                  A proximité :
                </h3>
                <ul>
                  <li>
                    Gare TGV 
                    <span
                      className="oi oi-arrow-thick-right"
                    />
                     Grenoble
                  </li>
                  <li>
                    Aéroport 
                    <span
                      className="oi oi-arrow-thick-right"
                    />
                     Lyon-Saint-Ex
                  </li>
                  <li>
                    Autoroutes 
                    <span
                      className="oi oi-arrow-thick-right"
                    />
                     A48 et A49
                  </li>
                </ul>
              </WrapperContent>
            </WrapperContent>,
            <WrapperContent />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <WrapperContent>
                  <h3>
                    Adresse :
                  </h3>
                  <ul
                    className="adress"
                  >
                    <li>
                      Domaine St-Jean de Chépy
                    </li>
                    <li>
                      11 Chemin du Domaine
                    </li>
                    <li>
                      38210 Tullins
                    </li>
                  </ul>
                </WrapperContent>,
                <WrapperContent>
                  <h3>
                    A proximité :
                  </h3>
                  <ul>
                    <li>
                      Gare TGV 
                      <span
                        className="oi oi-arrow-thick-right"
                      />
                       Grenoble
                    </li>
                    <li>
                      Aéroport 
                      <span
                        className="oi oi-arrow-thick-right"
                      />
                       Lyon-Saint-Ex
                    </li>
                    <li>
                      Autoroutes 
                      <span
                        className="oi oi-arrow-thick-right"
                      />
                       A48 et A49
                    </li>
                  </ul>
                </WrapperContent>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    <h3>
                      Adresse :
                    </h3>,
                    <ul
                      className="adress"
                    >
                      <li>
                        Domaine St-Jean de Chépy
                      </li>
                      <li>
                        11 Chemin du Domaine
                      </li>
                      <li>
                        38210 Tullins
                      </li>
                    </ul>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Adresse :",
                    },
                    "ref": null,
                    "rendered": "Adresse :",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <li>
                          Domaine St-Jean de Chépy
                        </li>,
                        <li>
                          11 Chemin du Domaine
                        </li>,
                        <li>
                          38210 Tullins
                        </li>,
                      ],
                      "className": "adress",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Domaine St-Jean de Chépy",
                        },
                        "ref": null,
                        "rendered": "Domaine St-Jean de Chépy",
                        "type": "li",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "11 Chemin du Domaine",
                        },
                        "ref": null,
                        "rendered": "11 Chemin du Domaine",
                        "type": "li",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "38210 Tullins",
                        },
                        "ref": null,
                        "rendered": "38210 Tullins",
                        "type": "li",
                      },
                    ],
                    "type": "ul",
                  },
                ],
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    <h3>
                      A proximité :
                    </h3>,
                    <ul>
                      <li>
                        Gare TGV 
                        <span
                          className="oi oi-arrow-thick-right"
                        />
                         Grenoble
                      </li>
                      <li>
                        Aéroport 
                        <span
                          className="oi oi-arrow-thick-right"
                        />
                         Lyon-Saint-Ex
                      </li>
                      <li>
                        Autoroutes 
                        <span
                          className="oi oi-arrow-thick-right"
                        />
                         A48 et A49
                      </li>
                    </ul>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "A proximité :",
                    },
                    "ref": null,
                    "rendered": "A proximité :",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <li>
                          Gare TGV 
                          <span
                            className="oi oi-arrow-thick-right"
                          />
                           Grenoble
                        </li>,
                        <li>
                          Aéroport 
                          <span
                            className="oi oi-arrow-thick-right"
                          />
                           Lyon-Saint-Ex
                        </li>,
                        <li>
                          Autoroutes 
                          <span
                            className="oi oi-arrow-thick-right"
                          />
                           A48 et A49
                        </li>,
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            "Gare TGV ",
                            <span
                              className="oi oi-arrow-thick-right"
                            />,
                            " Grenoble",
                          ],
                        },
                        "ref": null,
                        "rendered": Array [
                          "Gare TGV ",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {
                              "className": "oi oi-arrow-thick-right",
                            },
                            "ref": null,
                            "rendered": null,
                            "type": "span",
                          },
                          " Grenoble",
                        ],
                        "type": "li",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            "Aéroport ",
                            <span
                              className="oi oi-arrow-thick-right"
                            />,
                            " Lyon-Saint-Ex",
                          ],
                        },
                        "ref": null,
                        "rendered": Array [
                          "Aéroport ",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {
                              "className": "oi oi-arrow-thick-right",
                            },
                            "ref": null,
                            "rendered": null,
                            "type": "span",
                          },
                          " Lyon-Saint-Ex",
                        ],
                        "type": "li",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            "Autoroutes ",
                            <span
                              className="oi oi-arrow-thick-right"
                            />,
                            " A48 et A49",
                          ],
                        },
                        "ref": null,
                        "rendered": Array [
                          "Autoroutes ",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {
                              "className": "oi oi-arrow-thick-right",
                            },
                            "ref": null,
                            "rendered": null,
                            "type": "span",
                          },
                          " A48 et A49",
                        ],
                        "type": "li",
                      },
                    ],
                    "type": "ul",
                  },
                ],
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <PageTitle
            title="Saint-Jean de Chépy"
          />,
          <WrapperContent>
            <WrapperContent>
              <WrapperContent>
                <h3>
                  Adresse :
                </h3>
                <ul
                  className="adress"
                >
                  <li>
                    Domaine St-Jean de Chépy
                  </li>
                  <li>
                    11 Chemin du Domaine
                  </li>
                  <li>
                    38210 Tullins
                  </li>
                </ul>
              </WrapperContent>
              <WrapperContent>
                <h3>
                  A proximité :
                </h3>
                <ul>
                  <li>
                    Gare TGV 
                    <span
                      className="oi oi-arrow-thick-right"
                    />
                     Grenoble
                  </li>
                  <li>
                    Aéroport 
                    <span
                      className="oi oi-arrow-thick-right"
                    />
                     Lyon-Saint-Ex
                  </li>
                  <li>
                    Autoroutes 
                    <span
                      className="oi oi-arrow-thick-right"
                    />
                     A48 et A49
                  </li>
                </ul>
              </WrapperContent>
            </WrapperContent>
            <WrapperContent />
          </WrapperContent>,
        ],
        "className": "Sjc",
        "id": "Sjc",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "title": "Saint-Jean de Chépy",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <WrapperContent>
                <WrapperContent>
                  <h3>
                    Adresse :
                  </h3>
                  <ul
                    className="adress"
                  >
                    <li>
                      Domaine St-Jean de Chépy
                    </li>
                    <li>
                      11 Chemin du Domaine
                    </li>
                    <li>
                      38210 Tullins
                    </li>
                  </ul>
                </WrapperContent>
                <WrapperContent>
                  <h3>
                    A proximité :
                  </h3>
                  <ul>
                    <li>
                      Gare TGV 
                      <span
                        className="oi oi-arrow-thick-right"
                      />
                       Grenoble
                    </li>
                    <li>
                      Aéroport 
                      <span
                        className="oi oi-arrow-thick-right"
                      />
                       Lyon-Saint-Ex
                    </li>
                    <li>
                      Autoroutes 
                      <span
                        className="oi oi-arrow-thick-right"
                      />
                       A48 et A49
                    </li>
                  </ul>
                </WrapperContent>
              </WrapperContent>,
              <WrapperContent />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <WrapperContent>
                    <h3>
                      Adresse :
                    </h3>
                    <ul
                      className="adress"
                    >
                      <li>
                        Domaine St-Jean de Chépy
                      </li>
                      <li>
                        11 Chemin du Domaine
                      </li>
                      <li>
                        38210 Tullins
                      </li>
                    </ul>
                  </WrapperContent>,
                  <WrapperContent>
                    <h3>
                      A proximité :
                    </h3>
                    <ul>
                      <li>
                        Gare TGV 
                        <span
                          className="oi oi-arrow-thick-right"
                        />
                         Grenoble
                      </li>
                      <li>
                        Aéroport 
                        <span
                          className="oi oi-arrow-thick-right"
                        />
                         Lyon-Saint-Ex
                      </li>
                      <li>
                        Autoroutes 
                        <span
                          className="oi oi-arrow-thick-right"
                        />
                         A48 et A49
                      </li>
                    </ul>
                  </WrapperContent>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": Array [
                      <h3>
                        Adresse :
                      </h3>,
                      <ul
                        className="adress"
                      >
                        <li>
                          Domaine St-Jean de Chépy
                        </li>
                        <li>
                          11 Chemin du Domaine
                        </li>
                        <li>
                          38210 Tullins
                        </li>
                      </ul>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Adresse :",
                      },
                      "ref": null,
                      "rendered": "Adresse :",
                      "type": "h3",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          <li>
                            Domaine St-Jean de Chépy
                          </li>,
                          <li>
                            11 Chemin du Domaine
                          </li>,
                          <li>
                            38210 Tullins
                          </li>,
                        ],
                        "className": "adress",
                      },
                      "ref": null,
                      "rendered": Array [
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "Domaine St-Jean de Chépy",
                          },
                          "ref": null,
                          "rendered": "Domaine St-Jean de Chépy",
                          "type": "li",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "11 Chemin du Domaine",
                          },
                          "ref": null,
                          "rendered": "11 Chemin du Domaine",
                          "type": "li",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "38210 Tullins",
                          },
                          "ref": null,
                          "rendered": "38210 Tullins",
                          "type": "li",
                        },
                      ],
                      "type": "ul",
                    },
                  ],
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": Array [
                      <h3>
                        A proximité :
                      </h3>,
                      <ul>
                        <li>
                          Gare TGV 
                          <span
                            className="oi oi-arrow-thick-right"
                          />
                           Grenoble
                        </li>
                        <li>
                          Aéroport 
                          <span
                            className="oi oi-arrow-thick-right"
                          />
                           Lyon-Saint-Ex
                        </li>
                        <li>
                          Autoroutes 
                          <span
                            className="oi oi-arrow-thick-right"
                          />
                           A48 et A49
                        </li>
                      </ul>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "A proximité :",
                      },
                      "ref": null,
                      "rendered": "A proximité :",
                      "type": "h3",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          <li>
                            Gare TGV 
                            <span
                              className="oi oi-arrow-thick-right"
                            />
                             Grenoble
                          </li>,
                          <li>
                            Aéroport 
                            <span
                              className="oi oi-arrow-thick-right"
                            />
                             Lyon-Saint-Ex
                          </li>,
                          <li>
                            Autoroutes 
                            <span
                              className="oi oi-arrow-thick-right"
                            />
                             A48 et A49
                          </li>,
                        ],
                      },
                      "ref": null,
                      "rendered": Array [
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": Array [
                              "Gare TGV ",
                              <span
                                className="oi oi-arrow-thick-right"
                              />,
                              " Grenoble",
                            ],
                          },
                          "ref": null,
                          "rendered": Array [
                            "Gare TGV ",
                            Object {
                              "instance": null,
                              "key": undefined,
                              "nodeType": "host",
                              "props": Object {
                                "className": "oi oi-arrow-thick-right",
                              },
                              "ref": null,
                              "rendered": null,
                              "type": "span",
                            },
                            " Grenoble",
                          ],
                          "type": "li",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": Array [
                              "Aéroport ",
                              <span
                                className="oi oi-arrow-thick-right"
                              />,
                              " Lyon-Saint-Ex",
                            ],
                          },
                          "ref": null,
                          "rendered": Array [
                            "Aéroport ",
                            Object {
                              "instance": null,
                              "key": undefined,
                              "nodeType": "host",
                              "props": Object {
                                "className": "oi oi-arrow-thick-right",
                              },
                              "ref": null,
                              "rendered": null,
                              "type": "span",
                            },
                            " Lyon-Saint-Ex",
                          ],
                          "type": "li",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": Array [
                              "Autoroutes ",
                              <span
                                className="oi oi-arrow-thick-right"
                              />,
                              " A48 et A49",
                            ],
                          },
                          "ref": null,
                          "rendered": Array [
                            "Autoroutes ",
                            Object {
                              "instance": null,
                              "key": undefined,
                              "nodeType": "host",
                              "props": Object {
                                "className": "oi oi-arrow-thick-right",
                              },
                              "ref": null,
                              "rendered": null,
                              "type": "span",
                            },
                            " A48 et A49",
                          ],
                          "type": "li",
                        },
                      ],
                      "type": "ul",
                    },
                  ],
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
